pkg/services/apiserver/appinstaller/installer.go

```diff
@@ -192,6 +192,7 @@ func createPostStartHook(
 		logger := logging.FromContext(hookContext.Context)
 		logger.Debug("Initializing app", "app", installer.ManifestData().AppName)
 
+		hookContext.LoopbackClientConfig.APIPath = "/apis"
```

Comment:
this is still not fixed by default, isn't it?

Reply:
removed. it's included in the current app SDK version